src/components/page-drawer.svelte

```diff
@@ -29,11 +36,17 @@
     on:click={drawer.close}
   />
   <div
-    class="fixed bottom-2 left-2 right-2 top-2 z-50 overflow-y-auto scroll-smooth rounded-lg bg-background shadow-2xl md:left-auto md:w-3/4 lg:w-2/3"
+    on:scroll={handleScroll}
+    class="fixed bottom-2 left-2 right-2 top-2 transition z-50 overflow-y-auto scroll-smooth rounded-lg bg-background shadow-2xl md:left-auto md:w-3/4 lg:w-2/3"
     in:fly={{ x: 300, duration: 200 }}
     out:fly={{ x: expanding ? -300 : 300, duration: expanding ? 200 : 100 }}
   >
-    <header class="z-50 flex items-center justify-between p-2">
+    <header
+      class={clsx(
+        'sticky bg-background top-0 left-0 z-50 flex items-center justify-between p-2',
+        isSticky && 'border-b-background-offset border-b'
```

Comment:
```suggestion
        'sticky bg-background top-0 left-0 z-50 flex items-center justify-between p-2 border-b border-b-transparent motion-safe:transition-colors',
        isSticky && 'border-b-background-offset'
```

Reply:
This doesn’t work 🤔 


https://github.com/user-attachments/assets/6171de4a-6537-4c50-b188-97711b497a9e